### PR TITLE
Fix virtualized environment

### PR DESCRIPTION
**Pull request by chess-knight**
Related to #395

### CODE REVIEW

File: docs/user-guide/src/quick-start.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ virt-install \
   --name bmh-vm-01 \
   --description "Virtualized BareMetalHost" \
   --osinfo=ubuntu-lts-latest \
-  --ram=4096 \
+  --ram=4096 `# use 8192 for Scenario 2` \
```
Comment (tuminoid) [inline docs/user-guide/src/quick-start.md:110]:
```suggestion
  --ram=4096  \
```
Put the comment on L105?

**Review by tuminoid — COMMENTED** [review]
/ok-to-test

File: docs/user-guide/src/quick-start.md (hunk 1 above)
Comment (chess-knight) [reply docs/user-guide/src/quick-start.md]: See d0d975cd9ee125f261f963332e870f2844119f63

**Comment by tuminoid** [comment]
Please squash commits, then we are good to merge.

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Review by lentzi90 — COMMENTED** [review]
/approve

**Pull request merged**